python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1453,8 +1453,8 @@ def get_asset_records(
         return self._event_storage.get_asset_records(asset_keys)
 
     @traced
-    def get_asset_event_tags(self, asset_key: AssetKey) -> Sequence[Tuple[str, AbstractSet[str]]]:
-        return self._event_storage.get_asset_event_tags(asset_key)
+    def get_all_event_tags_for_asset(self, asset_key: AssetKey) -> Sequence[Tuple[str, AbstractSet[str]]]:
```

Comment:
might be okay for now, but we might want to add an optional filter arg here, so that we can fix one of the partition dimension values

Reply:
Sounds good! This method is now updated to contain this filter arg and also filters out wiped materializations